pack.ps1

```diff
@@ -27,9 +28,9 @@ function createNuGetPackage([string]$pkgName, [string]$culture)
     $cultureFolder = [IO.Path]::Combine($localizationFolderName, $culture)
     Copy-Item -Path $cultureFolder -Destination $pkgCultureFolderPath -Recurse
     
-    echo "Copying '.props' template file .."
+    echo "Copying '$pkgName.$pkgPropsExtension' ..."
 
-    $pkgBuildFolderPath = [IO.Path]::Combine($pkgFolderPath, "buildTransitive")
+    $pkgBuildFolderPath = [IO.Path]::Combine($pkgFolderPath, $pkgBuildFolderName)
```

Comment:
Seems you did before you saw my comment, coz there's no need for this change while it appears once

It's my fault during review, sorry

Reply:
no problem